daprdocs/content/en/developing-applications/building-blocks/service-invocation/howto-invoke-discover-services.md

```diff
@@ -1,8 +1,8 @@
 ---
 type: docs
-title: "How-To: Invoke and discover services"
+title: "How-To: Invoke services using HTTP"
```

Comment:
should a gRPC how to also be added?

Reply:
Yes, it would be to have an gRPC one.

Reply:
@orizohar @AaronCrawfis @yaron2 @mukundansundar  - can I get an approval here from someone? Thanks

Reply:
Raise this issue to track the gRPC suggestion. https://github.com/dapr/docs/issues/1215